packages/desktop/components/popup/popups/EvmTransactionFromDappPopup.svelte

```diff
@@ -63,6 +67,7 @@
                 break
             }
             default: {
+                isSmartContractCall = !!preparedTransaction.data
```

Comment:
We will need a more ellaborate check, because the protocol allows sending data to an EOA. A better check is to see whether the address contains bytecode on the EVM blockchain using the explorer API. I'll create a separate task for this.